ios/RNSScreenStackHeaderConfig.h

```diff
@@ -19,10 +19,9 @@
 #endif
 
 @property (nonatomic, weak) RNSScreenView *screenView;
+@property (nonatomic) NSMutableArray<RNSScreenStackHeaderSubview *> *reactSubviews;
```

Comment:
Why did `reactSubviews` land here in the end? Can't they be managed in implementation file like it was on paper?

Reply:
I think it can. However we're exposing getter to this prop any way so I don't see much difference if this prop is exposed in interface or not.

Reply:
I made it private: [752987e](https://github.com/software-mansion/react-native-screens/pull/1418/commits/752987e82d88d052da156179424f16f838e623a7)